pkgs/development/libraries/opendht/default.nix

```
@@ -20,15 +20,15 @@
   enablePushNotifications ? false,
 }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "opendht";
-  version = "3.2.0-unstable-2025-01-05";
+  version = "3.3.1rc2-unstable-2025-04-15";
 
   src = fetchFromGitHub {
     owner = "savoirfairelinux";
     repo = "opendht";
-    rev = "5237f0a3b3eb8965f294de706ad73596569ae1dd";
-    hash = "sha256-qErVKyZQR/asJ8qr0sRDaXZ8jUV7RaSLnJka5baWa7Q=";
+    rev = "2c26030108ede63e74efcc2d9c23ad47fdc8372e";
+    hash = "sha256-sjcjKj5veWIsO8JR6v17YyHZNB0K60ZAVqNBpin6HfI=";
```

This hash doesn't work, here is the correct one (according to the CI):

```suggestion
    hash = "sha256-xATRoZg18ZIKqT3TUDdcJXl5jFHz8Xo7NY7K1c39uRg=";
```

---

Ah, I guess I changed the version but forgot to update the hash... Thanks!

---

The new commit builds on my machine, thanks!